AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -693,18 +710,18 @@ - (void)authTokenContext:(MSAuthTokenContext *)__unused authTokenContext didUpda
 
 #pragma mark - Reachability
 
-- (void)networkStateChanged:(NSNotificationCenter *)__unused notification {
+- (void)networkStateChangedDataStoreCallback:(NSNotificationCenter *)__unused notification {
```

Comment:
The old name `networkStateChanged:` was OK to me, that's what we do in `MSHttpClient.m`.